### PR TITLE
docs: add links for commands

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
docs

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
Yes
**Summary**

 add links for `help` and `version` command

**Does this PR introduce a breaking change?**
No

**Other information**
No

### CODE REVIEW

**Review by evenstensberg — DISMISSED** [review]

**Review by alexander-akait — COMMENTED** [review]
Maybe we can add link to all commands?

**Comment by snitin315** [comment]
I will document the remaining commands on webpack.js.org and add links here 👍

File: README.md [hunk 1]
```diff
@@ -56,17 +56,17 @@ We organize webpack CLI as a multi-package repository using [lerna](https://gith
 Supporting developers is an important task for webpack CLI.
 Thus, webpack CLI provides different commands for many common tasks.
 
-- `build|bundle|b [entries...] [options]` - Run webpack (default command, can be omitted).
+- [`build|bundle|b [entries...] [options]`](https://webpack.js.org/api/cli/#build) - Run webpack (default command, can be omitted).
 - [`configtest|t [config-path]`](./packages/configtest/README.md#webpack-cli-configtest) - Validate a webpack configuration.
-- `help|h [command] [option]` - Display help for commands and options.
+- [`help|h [command] [option]`](https://webpack.js.org/api/cli/#help) - Display help for commands and options.
 - [`init|create|new|c|n [generation-path] [options]`](./packages/generators/INIT.md#webpack-cli-init) - Create a new webpack project.
 - [`info|i [options]`](./packages/info/README.md#webpack-cli-info) - Returns information related to the local environment.
```
Comment (alexander-akait) [inline README.md:63]: Maybe this too? Ideally we should provide link on all commands

Comment (snitin315) [reply README.md:63]: They are actually linked to the package README, but I think links to webpack.js.org will be better. I will update 👍🏻

File: README.md [hunk 2]
```diff
@@ -57,14 +57,14 @@ Supporting developers is an important task for webpack CLI.
...
+- [`init|create|new|c|n [generation-path] [options]`](https://webpack.js.org/api/cli/#init) - Create a new webpack project.
+- [`info|i [options]`](https://webpack.js.org/api/cli/#info) - Returns information related to the local environment.
 - [`migrate|m <config-path> [new-config-path]`](https://www.npmjs.com/package/@webpack-cli/migrate) - Migrate project from one version to another.
-- [`plugin|p [output-path] [options]`](./packages/generators#generators) - Initiate new plugin project.
-- [`loader|l [output-path] [options]`](./packages/generators#generators) - Initiate new loader project.
-- [`serve|server|s [entries...] [options]`](./packages/serve/README.md#webpack-cli-serve) - Use webpack with a development server that provides live reloading.
+- [`plugin|p [output-path] [options]`](https://webpack.js.org/api/cli/#plugin) - Initiate new plugin project.
+- [`loader|l [output-path] [options]`](https://webpack.js.org/api/cli/#loader) - Initiate new loader project.
+- [`serve|server|s [entries...] [options]`](https://webpack.js.org/configuration/dev-server/) - Use webpack with a development server that provides live reloading.
```
Comment (alexander-akait) [inline README.md:67]: I think we should have section for serve CLI too, right now we have link on options, it is bit weird

**Review by alexander-akait — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -56,17 +56,17 @@ We organize webpack CLI as a multi-package repository using [lerna](https://gith
...
-- [`loader|l [output-path] [options]`](./packages/generators#generators) - Initiate new loader project.
-- [`serve|server|s [entries...] [options]`](./packages/serve/README.md#webpack-cli-serve) - Use webpack with a development server that provides live reloading.
-- `version|v [commands...]` - Output the version number of `webpack`, `webpack-cli`, `webpack-dev-server`, and commands
-- `watch|w [entries...] [options]` - Run webpack and watch for files changes.
+- [`plugin|p [output-path] [options]`](https://webpack.js.org/api/cli/#plugin) - Initiate new plugin project.
+- [`loader|l [output-path] [options]`](https://webpack.js.org/api/cli/#loader) - Initiate new loader project.
+- [`serve|server|s [entries...] [options]`](https://webpack.js.org/api/cli/#serve) - Use webpack with a development server that provides live reloading.
+- [`version|v [commands...]`](https://webpack.js.org/api/cli/#version) - Output the version number of `webpack`, `webpack-cli`, `webpack-dev-server`, and commands.
+- [`watch|w [entries...] [options]`](https://webpack.js.org/api/cli/#watch) - Run webpack and watch for files changes.
```
Comment (alexander-akait) [inline README.md:69]: hm, mabe we can add two links, on docs page and readme page, so each command will have official doc and docs in own repo?

Comment (snitin315) [reply README.md:69]: `version`, `help` and `watch` don't have their own readme. Also, `init`, `plugin` and `loader` share a [common readme](https://github.com/webpack/webpack-cli/tree/master/packages/generators) which is not so informative and redirects to [actual code](https://github.com/webpack/webpack-cli/tree/master/packages/generators#generators).

Only `info`, `serve`, and `configtest` seem to have a proper readme page.

Comment (rishabh3112) [reply README.md:69]: > hm, mabe we can add two links, on docs page and readme page, so each command will have official doc and docs in own repo?

I think we should have single source of truth here on docs website itself 😅 .

Comment (snitin315) [reply README.md:69]: Yes, I think the same.

**Review by alexander-akait — APPROVED** [review]

**Review by rishabh3112 — APPROVED** [review]

**Pull request merged**